cli/src/main/kotlin/StartCommand.kt

```diff
@@ -45,13 +45,13 @@ internal val POLL_INTERVAL = System.getProperty("POLL_INTERVAL")?.toLongOrNull()
 class StartCommand : SuspendingCliktCommand(name = "start") {
     private val repositoryId by option(
         "--repository-id",
-        envvar = "REPOSITORY_ID",
+        envvar = "ORT_RUNS_START_REPOSITORY_ID",
```

Comment:
I find the `ORT_RUNS_` prefix a bit weird. I'm aware that other commands already use it, but I guess all commands should align on not using `ORT_` at all, but use `OSC_` instead as a prefix.